p5/io/local_data.py

```diff
@@ -0,0 +1,44 @@
+class table:
+
+	def __init__(self,PATH):
+		try:
+			file = open(PATH,'r')
+		except:
```

Comment:
I think it's ok to not catch the error here and simply let upstream code handle it.